pydash/pydash_app/dashboard/endpoint.py

```diff
@@ -20,9 +21,11 @@ def __init__(self, name, is_monitored):
 
         self.id = uuid.uuid4()
         self.name = name
-        self.calls = list()
         self.is_monitored = is_monitored
 
+        self._calls = []
+        self._aggregator = Aggregator(self._calls)
```

Comment:
Same as with dashboard.py: having a public function for extracting the aggregator data would be helpful.

Reply:
Yep, will add this.